pysass.cpp

```diff
@@ -512,19 +512,24 @@ PySass_compile_string(PyObject *self, PyObject *args) {
     struct Sass_Context *ctx;
     struct Sass_Data_Context *context;
     struct Sass_Options *options;
-    char *string, *include_paths;
+    char *string, *include_paths, *source_map_file;
     const char *error_message, *output_string;
     Sass_Output_Style output_style;
-    int source_comments, error_status, precision, indented;
+    int source_comments, error_status, precision, indented,
+        source_map_embed, source_map_contents, source_map_file_urls,
+        omit_source_map_url;
     PyObject *custom_functions;
     PyObject *custom_importers;
+    PyObject *source_map_root;
     PyObject *result;
 
     if (!PyArg_ParseTuple(args,
-                          PySass_IF_PY3("yiiyiOiO", "siisiOiO"),
+                          PySass_IF_PY3("yiiyiiOO|iiiO", "siisiiOO|iiiO"),
```

Comment:
iirc `|` is optional? let's make these always required -- the `.so` api isn't public and if we can control all the inputs we should

Reply:
Yes, `|` is optional. I was being conservative, not knowing whether or not this interface was considered public. Happy to make these fields required (likewise for the comment about `compile_dirname`)

Reply:
thanks!